hermes-ffi/src/ontology/injection.rs

```diff
@@ -386,6 +386,72 @@ impl AsRust<hermes::InjectionResetCompleteMessage> for CInjectionResetCompleteMe
     }
 }
 
+#[repr(C)]
+#[derive(Debug)]
+pub struct CInjectionFailedMessage {
+    pub request_id: *const libc::c_char,
+    pub context: *const libc::c_char,
+}
+
+unsafe impl Sync for CInjectionFailedMessage {}
+
+impl Drop for CInjectionFailedMessage {
+    fn drop(&mut self) {
+        take_back_nullable_c_string!(self.request_id);
+        take_back_nullable_c_string!(self.context);
+    }
+}
+
+impl CReprOf<hermes::InjectionFailedMessage> for CInjectionFailedMessage {
+    fn c_repr_of(message: hermes::InjectionFailedMessage) -> Fallible<Self> {
+        Ok(Self {
+            request_id: convert_to_nullable_c_string!(message.request_id),
+            context: convert_to_nullable_c_string!(message.context),
+        })
+    }
+}
+
+impl AsRust<hermes::InjectionFailedMessage> for CInjectionFailedMessage {
+    fn as_rust(&self) -> Fallible<hermes::InjectionFailedMessage> {
+        let request_id = create_optional_rust_string_from!(self.request_id);
+        let context = create_optional_rust_string_from!(self.context);
+        Ok(hermes::InjectionFailedMessage { request_id, context })
+    }
+}
+
+#[repr(C)]
+#[derive(Debug)]
+pub struct CInjectionResetFailedMessage {
+    pub request_id: *const libc::c_char,
```

Comment:
this field is nullable, add at least a `/// nullable` comment on it so that this appears in the generated .h